docker-compose.yml

```diff
@@ -2,6 +2,7 @@ version: '2.3'
 services:
  pgwatch:
   image: cybertec/pgwatch2:$BUILDTAG
+  restart: always
```

Comment:
Why `always` cos I know we normally use `unless-stopped`? Is there a specific reason for this change?

Reply:
@MusaZwane - here